sky/skylet/log_lib.py

```diff
@@ -2,6 +2,7 @@
 
 This is a remote utility module that provides logging functionality.
 """
+from collections import deque
```

Comment:
Import modules instead of class. See reference: https://google.github.io/styleguide/pyguide.html#22-imports